ts/routes/deck-options/SimulatorModal.svelte

```diff
@@ -0,0 +1,387 @@
+                    <SettingTitle on:click={() => openHelpModal("simulateFsrsReview")}>
+                        {tr.schedulingMaximumInterval()}
+                    </SettingTitle>
+                </SpinBoxRow>
+
+                <EnumSelectorRow
+                    bind:value={simulateFsrsRequest.reviewOrder}
+                    defaultValue={$config.reviewOrder}
+                    choices={reviewOrderChoices($fsrs)}
+                >
+                    <SettingTitle on:click={() => openHelpModal("simulateFsrsReview")}>
+                        {tr.deckConfigReviewSortOrder()}
+                    </SettingTitle>
+                </EnumSelectorRow>
+
+                <SwitchRow
+                    bind:value={simulateFsrsRequest.newCardsIgnoreReviewLimit}
+                    defaultValue={$newCardsIgnoreReviewLimit}
+                >
+                    <SettingTitle on:click={() => openHelpModal("simulateFsrsReview")}>
+                        <GlobalLabel title={tr.deckConfigNewCardsIgnoreReviewLimit()} />
+                    </SettingTitle>
+                </SwitchRow>
+
+                <SwitchRow bind:value={smooth} defaultValue={true}>
+                    <SettingTitle on:click={() => openHelpModal("simulateFsrsReview")}>
+                        {"Smooth Graph"}
+                    </SettingTitle>
+                </SwitchRow>
+
+                <button
+                    class="btn {computing ? 'btn-warning' : 'btn-primary'}"
+                    disabled={computing}
+                    on:click={simulateFsrs}
+                >
+                    {tr.deckConfigSimulate()}
+                </button>
+
+                <button
+                    class="btn {computing ? 'btn-warning' : 'btn-primary'}"
+                    disabled={computing}
+                    on:click={clearSimulation}
+                >
+                    {tr.deckConfigClearLastSimulate()}
+                </button>
+
+                <button
+                    class="btn {computing ? 'btn-warning' : 'btn-primary'}"
+                    disabled={computing}
+                    on:click={() => {
+                        $config.newPerDay = simulateFsrsRequest.newLimit;
+                        $config.reviewsPerDay = simulateFsrsRequest.reviewLimit;
+                        $config.maximumReviewInterval = simulateFsrsRequest.maxInterval;
+                        $config.desiredRetention = simulateFsrsRequest.desiredRetention;
+                        $newCardsIgnoreReviewLimit =
+                            simulateFsrsRequest.newCardsIgnoreReviewLimit;
+                        $config.reviewOrder = simulateFsrsRequest.reviewOrder;
+                        onPresetChange();
+                    }}
+                >
+                    <!-- {tr.deckConfigApplyChanges()} -->
+                    {"Save to Preset Options"}
+                </button>
+
+                {#if simulating}
+                    {tr.qtMiscProcessing()}
```

Comment:
Didn't catch this one in the past - our graphs shouldn't refer to any Qt strings, as they're supposed to only be for the Qt UI. I'll dupe it into the core repo.

Reply:
Ok, you can now use `tr.actionsProcessing` after rebasing over main.